docs/guides/custom-server.md

```diff
@@ -15,20 +15,22 @@ handles the GraphQL API and Admin UI. Things such as:
 - Add additional routes
 - Setup additional server middleware (`compress`/`brotli`/etc)
 - Notify a 3rd party service when the API is ready
-- ... etc
+- ...
 
 A **Custom Server** can replace the default and act as the entry point to your
 application which consumes your [schema definition](/docs/guides/schema.md). A Custom
-Server must handle initialising a http server which correctly executes any given KeystoneJS Apps.
+Server must handle initializing a http server which correctly executes any given KeystoneJS Apps.
```

Comment:
```suggestion
Server must handle initialising a http server which correctly executes any given KeystoneJS Apps.
```

Reply:
I noticed, while reading the documentation, that indeed most was en-UK. New PRs will not contain corrections to en-US..